packages/admin-stories/src/docs/components/Table/Basics.stories.mdx

```diff
@@ -0,0 +1,86 @@
+import { Meta, Canvas, Story, Source } from "@storybook/addon-docs";
+
+import dedent from "ts-dedent";
+
+<Meta title="Docs/Components/Table/Basics" />
+
+# Table
+
+> Tables display information in a way that's easy to scan, so that users can look for patterns and insights.
+>
+> \- [MUI Docs](https://mui.com/components/tables/)
+
+<Canvas>
+    <Story id="stories-components-table-basic-table--basic-table" />
+</Canvas>
+
+`Table` has three required parameters: `data`, `totalCount` and `columns`.
+
+`data` contains the data that should be displayed in the table. It must be an array of objects, and each object must have a unique `id`.
+
+`totalCount` specifies the total number of items.
+It's only relevant if pagination is used. Then, the totalCount should be provided by the API.
+If **no pagination** is used, `totalCount` can simply be set to `data.length`.
```

Comment:
We could consider adding this as a default values:

```typescript
function Table({ data, totalCount = data.length }) {
  ...
}
```

Reply:
At the moment, `totalCount` is always required, but it's only used if pagination is used. Otherwise the value doesn't matter.

So we could also make `totalCount` optional and throw an error if pagination is used but there is no `totalCount`.